libcontainer/init_linux.go

```diff
@@ -46,6 +46,15 @@ type network struct {
 	TempVethPeerName string `json:"temp_veth_peer_name"`
 }
 
+type mountFds struct {
+	// Fds to use as source when mounting
```

Comment:
nit: the comment should start with `sourceFds is/are ...`

Reply:
Fixed, thanks.

We probably need a linter for these things